src/Compute/Compute/Extension/VmssDiskEncryption/SetAzureVmssDiskEncryptionExtension.cs

```diff
@@ -120,6 +128,18 @@ public class SetAzureVmssDiskEncryptionExtensionCommand : VirtualMachineScaleSet
         [ValidateNotNullOrEmpty]
         public string ExtensionName { get; set; }
 
+        [Parameter(
+           Mandatory = false,
+           ValueFromPipelineByPropertyName = true,
+           HelpMessage = "The extension type. Specify this parameter to override its default value of \"AzureDiskEncryption\" for Windows VMs and \"AzureDiskEncryptionForLinux\" for Linux VMs.")]
+        [ValidateNotNullOrEmpty]
+        public string ExtensionType { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "Encrypt-Format all data drives that are not already encrypted")]
```

Comment:
Remove the dash between Encrypt-Format so that the value will match (if a dash is provided it won't work) 

"Encrypt-Format all" -> "EncryptFormatAll"